cdk/src/open-data-platform/data-plane/schema/schema.sql

```diff
@@ -28,12 +28,27 @@ CREATE TABLE IF NOT EXISTS demographics(
 );
 
 ALTER TABLE demographics
-    ADD COLUMN IF NOT EXISTS geom GEOMETRY(Polygon);
+    -- SRID 4326 maps the shape to latitude and longitude.
+    ADD COLUMN IF NOT EXISTS geom GEOMETRY(MultiPolygon, 4326);
 
 CREATE INDEX IF NOT EXISTS geom_index
     ON demographics
     USING GIST (geom);
 
+CREATE TABLE IF NOT EXISTS lead_service_lines(
+    pws_id varchar(255) NOT NULL,
+    lead_connections_count real,
+    PRIMARY KEY(pws_id)
+    );
+
+ALTER TABLE lead_service_lines
+    -- SRID 4326 maps the shape to latitude and longitude.
+    ADD COLUMN IF NOT EXISTS geom GEOMETRY(MultiPolygon, 4326);
```

Comment:
Perhaps make this number a constant explaining SRID 4326 maps the shape to latitude and longitude?

Reply:
Good idea. Done.